plugins/aws/fix_plugin_aws/resource/s3.py

```diff
@@ -185,6 +186,9 @@ class AwsS3Bucket(AwsResource, BaseBucket):
     bucket_logging: Optional[AwsS3Logging] = field(default=None)
     bucket_location: Optional[str] = field(default=None)
 
+    def resource_policy(self, builder: GraphBuilder) -> List[Tuple[PolicySource, Dict[str, Any]]]:
+        return [(PolicySource.resource, self.bucket_policy)] if self.bucket_policy else []
```

Comment:
Note: I assume the s3 bucket is one of the most complex things. It defines:
- acls on the bucket
- IAM permissions
- it has simplified bucket settings via public_access_block_configuration
- you can define those settings on an account via aws_s3_account_settings

We only talk about IAM for the access edges —maybe we should name them IAM edges to make this more clear.